src/amqp1.c

```diff
@@ -308,9 +310,25 @@ static int encqueue(cd_message_t *cdm,
                     amqp1_config_instance_t *instance) /* {{{ */
 {
   /* encode message */
-  pn_message_t *message = pn_message();
+  pn_message_t *message;
```

Comment:
Hi,

Please define variables on first use
https://github.com/collectd/collectd/blob/master/docs/review_comments.md

Reply:
Ok, I will adjust that